app/src/main/java/kr/ac/kaist/team888/mylittlefont/MainActivity.java

```diff
@@ -419,4 +276,15 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
 
     dialog.show();
   }
+
+  private void aboutus() {
+    ImageView imageView = new ImageView(this);
+    imageView.setImageDrawable(getDrawable(R.drawable.about_us));
+    imageView.setScaleType(ImageView.ScaleType.CENTER);
```

Comment:
This option makes image be shown partially for small layout device.
```java
imageView.setScaleType(ImageView.ScaleType.CENTER_INSIDE);
imageView.setAdjustViewBounds(true);
```
would be the best (from [stackoverflow](https://stackoverflow.com/a/19316787/2868508))